dino/src/game/dinosaur/mod.rs

```diff
@@ -1,31 +1,33 @@
-use bevy::prelude::*;
-
-mod components;
+pub mod components;
```

Comment:
```suggestion
mod components;
```
Agora esse módulo não precisa ser público